google-startup-scripts/usr/lib/systemd/system/google-shutdown-scripts.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Google Compute Engine user shutdown scripts
+Before=shutdown.target reboot.target halt.target
+DefaultDependencies=true
+Wants=local-fs.target network-online.target network.target
+
+[Service]
+ExecStart=/usr/share/google/run-shutdown-scripts
+Type=oneshot
+
+[Install]
+WantedBy=shutdown.target reboot.target halt.target
```

Comment:
As I said you don't need to mention reboot.target and halt.target here. shutdown.target precedes them both.